.travis.yml

```diff
@@ -3,4 +3,16 @@ language: perl
 perl:
   - '5.18'
   - '5.24'
+services:
+  - postgresql
+addons:
+  postgresql: '9.4'
+before_script:
+  - sudo mkdir /etc/dochazka-rest
+  - sudo sh -c "echo \"set( 'DOCHAZKA_DBNAME', 'dochazka-test' );\" >> /etc/dochazka-rest/REST_SiteConfig.pm"
+  - sudo sh -c "echo \"set( 'DOCHAZKA_DBUSER', 'dochazka' );\" >> /etc/dochazka-rest/REST_SiteConfig.pm"
+  - sudo sh -c "echo \"set( 'DOCHAZKA_DBPASS', 'dochazka' );\" >> /etc/dochazka-rest/REST_SiteConfig.pm"
+  - chmod +x bin/*
```

Comment:
for deployments via git (testing or production at the tag). Modern web frameworks like rails do have executable scripts in bin/ as well